content/lua/concepts/strings/terms/find/find.md

```diff
@@ -0,0 +1,176 @@
+---
+Title: '.find()'
+Description: 'The **`.find()`** function is used to search for a substring within a string and returns the start and end indices of the first occurrence of the substring.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Functions'
+  - 'Index'
+  - 'Methods'
+  - 'Search'
+  - 'Strings'
+CatalogContent:
+  - 'learn-lua'
+  - 'paths/computer-science'
+---
+
+The **`.find()`** function is used to search for a substring within a string and returns the start and end indices of the first occurrence of the substring.
+
+If the substring is not found, the function returns `nil`.
```

Comment:
```suggestion
The **`.find()`** function is used to search for a substring within a string and returns the start and end indices of the first occurrence of the substring. If the substring is not found, the function returns `nil`.
```

Reply:
This makes sense, but because I was basing it off [this](https://github.com/Codecademy/docs/blob/main/content/python/concepts/strings/terms/find/find.md), it might be worth editing as well.